spec/features/badge_spec.rb

```diff
@@ -61,20 +49,58 @@ def log_user(login, password)
       check 'pref_issue_badge'
       click_on 'Save'
 
-      expect(page).to have_selector("#issue_badge")
+      expect(page).to have_selector('#issue_badge')
+    end
 
-      user = User.where(:login => 'dlopper').first
-      all_issues = Issue.visible.open.where(:assigned_to_id => ([user.id] + user.group_ids))
-      puts "dlopper's issue: #{all_issues.length}"
-      if all_issues.any?
-        expect(page).to have_selector("#issue_badge_number")
-        expect(page).to have_selector("#issue_badge_number"), text: all_issues.length
+    scenario 'Badge number is displayed if badge option is activated and operator has assigned issues.' do
+      # Enable Badge
+      check 'pref_issue_badge'
+      click_on 'Save'
 
-        puts "Badge count: " + find(:id, 'issue_badge_number').text
-        page.save_screenshot('capture/badge.png', :full => true)
+      user = User.where(login: 'dlopper').first
+      all_issues = Issue.visible.open.where(assigned_to_id: ([user.id] + user.group_ids))
+      if all_issues.any?
+        expect(page).to have_selector('#issue_badge_number'), text: all_issues.length
```

Comment:
__[rubocop]__
* Line is too long. [85/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)